<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QWidget" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../hiway.qrc">:/images/res/menu.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lblNum">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../hiway.qrc">:/images/res/num.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lblBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Hiway</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTime">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDate">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <zorder>lblBackground</zorder>
  <zorder>lblMenu</zorder>
  <zorder>lblNum</zorder>
  <zorder>label</zorder>
  <zorder>lblTime</zorder>
  <zorder>lblDate</zorder>
 </widget>
 <resources>
  <include location="../hiway.qrc"/>
 </resources>
 <connections/>
</ui>
